README.md

```diff
@@ -26,6 +26,48 @@ Make a image of OSM data of an area from 2 dates, showing what was changed.
    cd osm-mapping-party-before-after
    ```
 
+### Install & use via Docker image
+
+Alternatively, if you do not want to install the whole pipeline yourself, you can run this setup in a Docker container that is ready to use.
+We start, by cloning this repository:
+
+```bash
+git clone --recurse-submodules https://github.com/amandasaurus/osm-mapping-party-before-after
+cd osm-mapping-party-before-after
+   ```
+Then we build the image that is specified in the `Dockerfile`, picking a name for the image, e.g. `before_after_builder`.
+Depending on how you installed Docker, you might have to run `sudo` for each `docker` command.
+
+ ```bash
+docker build -t before_after_builder
+```
+
+This step needs to be run only once, and it can take a few minutes as it will download and build all the dependencies needed. 
+Once the step is finished, you can launch the container like so:
+
+```bash
+./docker_run.sh before_after_builder /full/path/to/in-out-dir/ 
+```
+
+The command takes two parameters:
+
+1. the name of the container (as specified above)
+2. a full path to a directory you want to use for accessing and writing files to from the container (i.e. Input and output files). Depending on how you installed/run Docker, this folder might need full read/write permissions for other users (e.g. run `chmod 777`). 
+
+The `docker_run.sh` command will launch the container itself, including the necessary postgres database etc.
+It will also create two virtual docker volumes (named `pgdata` and `osm_data`). 
+These volumes are used to store external data, i.e. the database contents and the `openstreetmap-carto` external files.
+
+Once the docker container is running, you can connect into it to get a bash inside to run the commands as outlined below using:
+
+```bash
+docker exec -ti map-before-after bash
+```
+
+The resulting shell puts you into the equivalent of the root of this repository, in a folder called `/workdir`. From there you can use this repository as outlined below. 
+
+All output files are saved in the `/workdir` by default, you can from there move them into `/workdir/output` inside the container to access files on your host operating system.
```

Comment:
This is quite painful for me to do in the command line. Couldn't this be done automatically for the result images?

Reply:
This could be done by amending the `make.sh`, but I didn't want to modify it too much without feedback from @amandasaurus for now!